app/controllers/accountants_controller.rb

```diff
@@ -5,28 +5,29 @@ class AccountantsController < ApplicationController
   before_action :find_patient, only: [:edit]
 
   def index
-    patients = pledged_patients.includes(:clinic)
-    @patients = paginate_results(patients)
+    @patients = paginate_results(pledged_patients)
+    @entry_name = t('common.patient').downcase
   end
 
   def search
     have_search = params[:search].present?
     have_clinic = params[:clinic_id].present?
 
-    if have_search || have_clinic
-      partial = Patient.where(pledge_sent: true)
-                       .includes(:clinic)
-                       .includes(:fulfillment)
-                       .order(pledge_sent_at: :desc)
+    @results = if have_search || have_clinic
```

Comment:
Nitpick: this is way slicker + totally the right way to do things, but may make the codebase slightly less beginner friendly. No strong opinions,

Reply:
fair enough! i only did this because the (rubo)cops told me 👮‍♀️